webui/react/src/pages/TrialDetails/TrialChart.tsx

```diff
@@ -29,36 +29,63 @@ const getChartMetricLabel = (metric: MetricName): string => {
   return metric.name;
 };
 
+const numerizeScale = (scale: Scale): number => {
+  return scale === Scale.Log ? 3 : 1;
+};
+
 const TrialChart: React.FC<Props> = ({
   defaultMetricNames,
   metricNames,
   metrics,
   onMetricChange,
-  workloads,
   trialId,
 }: Props) => {
   const [ scale, setScale ] = useState<Scale>(Scale.Linear);
+  const [ trialSumm, setTrialSummary ] = useState<MetricContainer[]>([]);
+
+  useMemo(async () => {
+    if (trialId) {
+      const summ = await getTrialSummary({
+        maxDatapoints: 30,
+        metricNames: metricNames,
+        scale: numerizeScale(scale),
+        trialId: trialId,
+      });
+      setTrialSummary(summ.metrics);
+    }
+  }, [ metricNames, trialId ]);
+
+  // const resetZoom = async (min: number, max: number) => {
+  //   if (trialId) {
+  //     const summ = await getTrialSummary({
+  //       endBatches: Math.ceil(max),
+  //       maxDatapoints: 30,
+  //       metricNames: metricNames,
+  //       startBatches: Math.floor(min),
+  //       trialId: trialId,
+  //     });
+  //     setTrialSummary(summ.metrics);
+  //   }
+  // };
```

Comment:
**chore:** drop, if it remains a comment

Reply:
👍  This resetZoom code is back